RealmSwift-swift2.0/Results.swift

```diff
@@ -132,7 +135,11 @@ public final class Results<T: Object>: ResultsBase {
     - returns: The index of the given object, or `nil` if no objects match.
     */
     public func indexOf(predicateFormat: String, _ args: AnyObject...) -> Int? {
-        return notFoundToNil(rlmResults.indexOfObjectWithPredicate(NSPredicate(format: predicateFormat, argumentArray: args)))
+        return notFoundToNil(
+            rlmResults.indexOfObjectWithPredicate(
+                NSPredicate(format: predicateFormat, argumentArray: args)
+            )
```

Comment:
The dangling paren on its own line is weird and not at all consistent with any of the other code.